.gitignore

```diff
@@ -2,3 +2,4 @@ dist
 node_modules
 build/report
 browserstack-run.pid
+temp/
```

Comment:
@ebenoist, it was discussed before about adding this temp folder on the gitignore. As it is only created during on grunt failures and it is not persistent, it shouldn't be here.

Although, I don't mind with it being listed here. If the team agrees, I would just leave this folder, better than having further PRs with the folder contents.